_component_design/left-nav.md

```diff
@@ -4,135 +4,168 @@ parent: Navigation
 layout: component
 category: Components
 usage: >
-  Left navigation establishes a user's location within an application and shows the path forward to accomplish a goal. Consistent navigation patterns increase user confidence and efficiency.
+  The left navigation pane provides a menu of navigation items that a user clicks to move to another page or location within the application they are using. Because it is easier for user to scan, left navigation is preferred over top navigation. Consistent navigation patterns increase user confidence and efficiency.
```

Comment:
nix "pane" throughout 
rationale: It's not a pane unless it can be collapses/dismissed.